packages/@guardian/source-react-components-development-kitchen/src/toggle-switch/styles.ts

```diff
@@ -106,6 +112,7 @@ export const webStyles = css`
 		border-bottom: 2px solid ${success[400]};
 		border-right: 2px solid ${success[400]};
 		transition: opacity 0.1s ease-in;
+		box-sizing: content-box;
```

Comment:
If we tweak the width and height of the the `:before`, we can achieve a similar effect without the need to override the `box-sizing`:

```css
height: 0.7rem;
width: 0.4rem;
```
**Chrome**
<img width="59" alt="Screenshot 2022-01-24 at 10 01 58" src="https://user-images.githubusercontent.com/5931528/150762233-549fdbe8-4fcc-47f3-b848-a75d4bdcfcaa.png">

**Safari**
<img width="60" alt="Screenshot 2022-01-24 at 10 02 24" src="https://user-images.githubusercontent.com/5931528/150762316-fa195f33-84bd-4ac0-9b1d-a712f015df45.png">


**Firefox**
<img width="62" alt="Screenshot 2022-01-24 at 10 02 49" src="https://user-images.githubusercontent.com/5931528/150762334-da0f64f7-2ddc-43fc-9f14-9461213f6311.png">


What do you think?

Reply:
Thanks, yeah that's much better! Have implemented and pushed changes here.